contracts/kernel/Kernel.sol

```diff
@@ -114,6 +114,10 @@ contract Kernel is IKernel, KernelStorage, Initializable, AppProxyFactory, ACLSy
     }
 
     function isContract(address _target) internal view returns (bool) {
+        if (_target == address(0)) {
```

Comment:
I think `extcodesize(0)` always produces 0 so this might not be necessary.

Reply:
`extcodesize` costs 700 gas and that check is almost free :)